coding-concepts.js

```diff
@@ -7,45 +7,47 @@
 
 // --------------------1) What will this log?
 
-var cohort = "Bravo 2021"
-// console.log(cohort.length)
-
-// a) Your answer:
-// b) Verify and explain:
+//var cohort = "Bravo 2021"
+ //console.log(cohort.length)
 
+// a) Your answer:10
+// b) Verify and explain
+//the console.log(cohort.lenght) is a built in method used to count the amount of characters in a variable/ string.
 
 // --------------------2) What will this log?
 
-var greeting = "Hello World!"
-// console.log(greeting[3])
+//var greeting = "Hello World!"
+//console.log(greeting[3])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: hello world hello world hellow world 
+// b) Verify and explain: it logged l becsuse it was refrencing the 3rd character in the string
 
 
 // --------------------3) What will this log?
 
-var languages = ["JavaScript", "Ruby", "Python", "C++"]
-var index = 1
-// console.log(languages[index])
+//var languages = ["JavaScript", "Ruby", "Python", "C++"]
+//var index = 1
+//console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: the answer will be ruby
+// b) Verify and explain: the answer will be ruby becausde index=1 is checking for string 1
 
 
 // --------------------4) What will this log?
 
-var weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+//var weekendDays = ["saturday", "sunday"]
+
+//console.log(weekendDays.toUpperCase())
+
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: it will change all the letters to uppercase 
+// b) Verify and explain: this didnt log because its not a proper functiomn 
 
```

Comment:
Yeah this is interesting because `.toUpperCase()` is valid but only on strings. It is being called on an array in this case.